### ISSUE TITLE
Why divide the loss by the epoch index?

### ISSUE THREAD

**Issue by 2jungeuni** [Mouradost/STGM#15]
First, thank you for releasing very well organized code. 

If you divide the loss by the epoch index, the loss will inevitably decrease as learning progresses.

Can you briefly explain the followings?
1. "self.metrics["train/loss"] += loss / (i + 1)" and "self.metrics["train/e_loss"] += e_loss / (i + 1)" in src/trainers/full.py
2. "self.metrics["val/loss"] += loss / (i + 1)" and "self.metrics["val/e_loss"] += e_loss / (i + 1)" in src/trainers/full.py
